export_variables.sh

```diff
@@ -18,16 +18,14 @@ if [ -z "$test_case" ] || [ "$test_case" = "container" ] ; then
   tmt_plan_suffix="-docker"
   context_suffix=""
   test_name="test"
-elif [ "$test_case" = openshift ] ; then
-  if [ "$os_test" = "centos7" ] || [ "$os_test" = "rhel7" ] ; then
+elif [ "$test_case" = "openshift" ] ; then
```

Comment:
I would suggest using `case`  instead of `if elif etc.`.

Reply:
Done